### PR TITLE
Suknuk badges

### PR DESCRIPTION
**Pull request by suknuk**
Adding the badges to the README.md file for issue #1

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Thanks, this is almost perfect!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 meta-search-engine
 ==================
 
+[![Build Status](https://travis-ci.org/schul-cloud/meta-search-engine.svg?branch=master)](https://travis-ci.org/schul-cloud/meta-search-engine)
+[![Docker Build Status](https://img.shields.io/docker/build/jrottenberg/ffmpeg.svg)](https://hub.docker.com/r/schulcloud/meta-search-engine/builds/)
```
Comment (niccokunzmann) [inline README.md:5]: In the docker line, you do not use the docker buils from schulcloud/meta-search-engine but from jrottenberg/ffmpeg
Apart from this, this is what we are looking for.

Comment (niccokunzmann) [reply README.md:5]: All the things you did "right" which could have gone wrong:
- use markdown, not html
- find out how to include images
- find the right images to include 1x
- use links 2x
:)

**Comment by suknuk** [comment]
Thanks for the feedback! I have created a new pull request with the (hopefully) correct link.

**Pull request merged**